### PR TITLE
Add documentation for the Elvia integration

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

Documentation for https://github.com/home-assistant/core/pull/107405

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/107405
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/5062
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/elvia.markdown [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: Elvia
+description: Documentation for the Elvia integration.
+ha_category:
+  - Energy
+ha_release: 9999.9 #placeholder version
```
Comment (ludeeus) [inline source/_integrations/elvia.markdown]: Placeholder version, needs replacement.

File: source/_integrations/elvia.markdown [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ha_iot_class: Cloud Polling
+ha_codeowners:
+  - '@ludeeus'
+ha_domain: elvia
+ha_config_flow: true
+ha_integration_type: integration
+---
+
+The Elvia integration will import you grid consumption.
```
Comment (frenck) [inline source/_integrations/elvia.markdown]:
```suggestion
The Elvia integration will import your grid consumption.
```

File: source/_integrations/elvia.markdown [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+You can use the Elvia integration, regardless of which provider you are subscribed to as long as Elvia operates the grid.
+
+The imported data can be used in the [Energy dashboard](/docs/energy/) and with the [statistics-graph card](/dashboards/statistics-graph/).
+
+## Setup
```
Comment (frenck) [inline source/_integrations/elvia.markdown]:
```suggestion
## Prerequisites
```

File: source/_integrations/elvia.markdown [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+To use this integration, you need to create an API token.
+
+1. Start by logging in to your [Min side](https://www.elvia.no/logg-inn/)
+2. Once you are logged in, select your main account (This will usually be the one with your name on it).
+3. Select the "Tilganger" tab.
+4. Under the "" section, select the "Opprett token for målerverdier i API"
+5. In the dialog select the meter(s) ([see note below about multiple meters](#multiple-meters-for-a-single-api-token)) you want to import values for, give it a name (like "Home Assistant integrasjon") and use the "Opprett" button to create your token.
+6. Copy the token shown in the dialog, and paste that in the integration configuration.
```
Comment (frenck) [inline source/_integrations/elvia.markdown]:
```suggestion
1. Start by logging in to your [Min side](https://www.elvia.no/logg-inn/).
2. Once logged in, select your main account (Usually the one with your name on it).
3. Select the **Tilganger** tab.
4. Under the "" section, select the **Opprett token for målerverdier i API**.
5. In the dialog, select the meter(s) ([see note below about multiple meters](#multiple-meters-for-a-single-api-token)) you want to import values for, give it a name (like "Home Assistant integrasjon") and use the **Opprett** button to create your token.
6. Copy the token shown in the dialog, and paste that in the integration configuration.
```

File: source/_integrations/elvia.markdown [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+{% include integrations/config_flow.md %}
+
+## Updates
+
+The integration will fetch new data every hour.
+The first time it runs the data for the past year will be imported.
```
Comment (frenck) [inline source/_integrations/elvia.markdown]:
```suggestion
The first time it runs, the data for the past year will be imported.
```

File: source/_integrations/elvia.markdown [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+## Multiple meters for a single API token
+
+While you can select multiple meters when you create your token, the integration will only allow 1 meter for each configuration of it.
+So if you are using the same token for multiple meters, you need to add the integration multiple times and selecting a different meter each time.
```
Comment (frenck) [inline source/_integrations/elvia.markdown]:
```suggestion
While you can select multiple meters when you create your token, the integration will only allow 1 meter for each configuration.
So if you are using the same token for multiple meters, you must add the integration multiple times and select a different meter each time.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by ludeeus** [comment]
Thanks @frenck !
> The ha_release might need an update before merging this one in.

might? 😂 

As its currently 9999.9 it definitely needs an update 😄

**Comment by frenck** [comment]
Yeah... there is quite a slight high probability it might maybe need an update... 😁

File: source/_integrations/elvia.markdown [hunk 7]
```diff
@@ -0,0 +1,45 @@
+---
+title: Elvia
+description: Documentation for the Elvia integration.
+ha_category:
+  - Energy
+ha_release: 9999.9 #placeholder version
```
Comment (edenhaus) [inline source/_integrations/elvia.markdown]:
```suggestion
ha_release: 2024.2
```

**Review by edenhaus — CHANGES_REQUESTED** [review]

**Review by edenhaus — APPROVED** [review]

**Pull request merged**